test/system/user_location_change_test.rb

```diff
@@ -0,0 +1,22 @@
+require "application_system_test_case"
+
+class UserLocationChangeTest < ApplicationSystemTestCase
+  def setup
+    stub_request(:get, /.*gravatar.com.*d=404/).to_return(:status => 404)
+  end
+
+  test "User can change their location" do
+    user = create(:user)
+    sign_in_as(user)
+
+    visit user_path(user.display_name)
```

Comment:
You write it differently below
```suggestion
    visit user_path(user)
```

Reply:
Was updated and removed to remain consistent in the test file.